examples/hosts/app-integration/external-data/src/mock-service/service.ts

```diff
@@ -114,14 +114,14 @@ export async function initializeCustomerService(props: ServiceProps): Promise<Se
         console.log(`SERVICE: Fetching task list data...`);
         externalDataSource.fetchData().then(
             (data) => {
-                console.log(`SERVICE: Returning current task list:\n"${data}".`);
-                result.send({taskList: data});
+                console.log(`SERVICE: Returning current task list:\n"${data.body}".`);
+                result.send(JSON.parse(data.body.toString()) as TaskData);
```

Comment:
Do we need to roundtrip this data through stringifying and re-parsing? Can we just forward the data unmodified?

Reply:
There are a couple of places in tests where we're doing this too. I'm curious if/why this is needed.